driver/git.go

```diff
@@ -53,35 +52,35 @@ func (driver *GitDriver) Bump(bump version.Bump) (semver.Version, error) {
 		return semver.Version{}, err
 	}
 
-	var newVersion semver.Version
+	lastError := fmt.Errorf("no more retries")
 
-	for i := 1; i <= RetriesOnErrorWriteVersion; i++ {
-		err = driver.setUpRepo()
-		if err != nil {
-			return semver.Version{}, err
+	for i := 0; i <= maxRetries; i++ {
```

Comment:
refer to https://github.com/concourse/semver-resource/pull/92#pullrequestreview-254751103 I think we still want the number to be 3 @vito ?